<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchProfWindow</class>
 <widget class="QMainWindow" name="SearchProfWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>400</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(108, 143, 196, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_search_profesor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string>Nombre o Apellido:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_listarProfesor">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 84, 166);
color: rgb(255, 255, 255);
border-radius:8px;
}
QPushButton:hover {
background-color:white;
color:rgb(0, 84, 166);
}
QPushButton:pressed  {
background-color: rgb(0, 84, 166);
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>Listar todos los profesores</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_search_profesor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:8px;
border: 1px solid gray;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search_profesor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>141</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 84, 166);
color: rgb(255, 255, 255);
border-radius:8px;
}
QPushButton:hover {
background-color:white;
color:rgb(0, 84, 166);
}
QPushButton:pressed  {
background-color: rgb(0, 84, 166);
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_profesor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>811</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>40</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color: rgba(0, 0, 0, 0.3); </string>
    </property>
    <property name="text">
     <string>BUSCAR PROFESORES</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back_alumno_search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 84, 166);
color: rgb(255, 255, 255);
border-radius:8px;
}
QPushButton:hover {
background-color:white;
color:rgb(0, 84, 166);
}
QPushButton:pressed  {
background-color: rgb(0, 84, 166);
color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QTableView" name="search_listView_profesor_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>801</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid black;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_userOn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: transparent;
color:rgb(0, 84, 166);
}</string>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
